InvenTree/order/serializers.py

```diff
@@ -335,13 +338,20 @@ def __init__(self, *args, **kwargs):
     part_detail = PartBriefSerializer(source='part', many=False, read_only=True)
     allocations = SalesOrderAllocationSerializer(many=True, read_only=True)
 
-    # TODO: Once https://github.com/inventree/InvenTree/issues/1687 is fixed, remove default values
-    quantity = serializers.FloatField(default=1)
+    quantity = serializers.FloatField()
 
     allocated = serializers.FloatField(source='allocated_quantity', read_only=True)
     fulfilled = serializers.FloatField(source='fulfilled_quantity', read_only=True)
+    
+    sale_price = InvenTreeMoneySerializer(max_digits=19, decimal_places=4)
+
     sale_price_string = serializers.CharField(source='sale_price', read_only=True)
 
+    sale_price_currency = serializers.ChoiceField(
+        choices=djmoney.settings.CURRENCY_CHOICES,
```

Comment:
@SchrodingersGat maybe use the new helper function `common.settings.currency_code_mappings` for this? Would provide support for setting choices in DB in the future and limit to accepted currencies.

Reply:
Thanks, I've taken that on board